src/main/java/com/impactupgrade/nucleus/client/MailchimpClient.java

```diff
@@ -115,9 +115,17 @@ public List<MemberInfo> getListMembers(String listId) throws IOException, Mailch
   }
 
   public List<MemberInfo> getListMembers(String listId, String status, Calendar sinceLastChanged) throws IOException, MailchimpException {
+    return getListMembers(listId, status, null, sinceLastChanged);
+  }
+
+  public List<MemberInfo> getListMembers(String listId, String status, String fields, Calendar sinceLastChanged) throws IOException, MailchimpException {
     GetMembersMethod getMembersMethod = new GetMembersMethod(listId);
     getMembersMethod.status = status;
-    getMembersMethod.fields = "members.email_address,members.tags,total_items"; // HUGE performance improvement -- limit to only what we need
+    if (!Strings.isNullOrEmpty(fields)) {
+      getMembersMethod.fields = fields;
+    } else {
+      getMembersMethod.fields = "members.email_address,members.tags,total_items"; // HUGE performance improvement -- limit to only what we need
+    }
```

Comment:
Simply do `getMembersMethod.fields = fields;` instead of the if/else block. Let the overloaded method (above) pass in the default?